connectrum/client.py

```diff
@@ -72,22 +82,25 @@ def close(self):
         hostname, port, use_ssl = server_info.get_port(proto_code)
 
         if use_tor:
-            # Connect via Tor proxy proxy, assumed to be on localhost:9050
-            # unless a tuple is given with another host/port combo.
-            try:
-                socks_host, socks_port = use_tor
-            except TypeError:
-                socks_host, socks_port = 'localhost', 9150
+            if have_aiosocks:
+                # Connect via Tor proxy proxy, assumed to be on localhost:9050
+                # unless a tuple is given with another host/port combo.
+                try:
+                    socks_host, socks_port = use_tor
+                except TypeError:
+                    socks_host, socks_port = 'localhost', 9150
 
-            # basically no-one has .onion SSL certificates, and
-            # pointless anyway.
-            disable_cert_verify = True
+                # basically no-one has .onion SSL certificates, and
```

Comment:
I've used 4 space tabs in both commits for increased indents. I've checked this line again in my editor, but it does not show a hard tab.